release-notes/10.0/preview/preview1/libraries.md

```diff
@@ -8,7 +8,173 @@
 
 - [What's new in .NET 10](https://learn.microsoft.com/dotnet/core/whats-new/dotnet-10/overview) documentation
 
+## Finding Certificates By Thumbprints Other Than SHA-1
 
-## Feature
+Finding certificates uniquely by thumbprint is a fairly common operation,
+but the `X509Certificate2Collection.Find` method (for the `FindByThumbprint` mode) only searches for the SHA-1 Thumbprint value.
+
+Since SHA-2-256 ("SHA256") and SHA-3-256 have the same lengths,
+we decided that it wasn't pure goodness to let the Find method find any vaguely matching thumbprints.
+Instead, we introduced a new method that accepts the name of the hash algorithm that you want to use for matching.
+
+```C#
+X509Certificate2Collection coll = store.Certificates.FindByThumbprint(HashAlgorithmName.SHA256, thumbprint);
+Debug.Assert(coll.Count < 2, "Collection has too many matches, has SHA-2 been broken?");
+return coll.SingleOrDefault();
+```
+
+## Finding PEM-encoded Data in ASCII/UTF-8
+
+The PEM encoding (originally "Privacy Enhanced Mail", but now used widely outside of email) is defined for "text",
+which means that the `PemEncoding` class was designed to run on `System.String` and `ReadOnlySpan<char>`.

```

Comment:
> which means that the `PemEncoding` class was designed to run

I'm not seeing the obvious conclusion that A leads to B.

Reply:
You don't think it's obvious that "text" meant "string"?

Reply:
OK. I get it now. Thanks.